migrations/Version20231006182551.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20231006182551 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE messenger_messages 
+        (id BIGINT AUTO_INCREMENT NOT NULL,
+        body LONGTEXT NOT NULL, headers LONGTEXT NOT NULL,
+        queue_name VARCHAR(190) NOT NULL,
+        created_at DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\',
+        available_at DATETIME NOT NULL COMMENT \'(DC2Type:datetime_immutable)\',
+        delivered_at DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\',
+        INDEX IDX_75EA56E0FB7336F0 (queue_name), INDEX IDX_75EA56E0E3BD61CE (available_at),
+        INDEX IDX_75EA56E016BA31DB (delivered_at), PRIMARY KEY(id))
+        DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('DROP TABLE messenger_messages');
+    }
+}
```

Comment:
No migrations file found for any of these entity.

Reply:
done